benchmarks/CMakeLists.txt

```diff
@@ -1,11 +1,22 @@
+if (Kokkos_ENABLE_OPENMPTARGET AND CMAKE_BUILD_TYPE!="Release" AND Kokkos_ENABLE_BENCHMARKS)
+    # FIXME_OPENMPTARGET - OpenMPTarget does not support debug symbols (Internal Compiler Error). Disabling benchmarks
+    message(WARNING "OpenMPTarget is enabled and CMAKE_BUILD_TYPE is not Release: disabling benchmarks to avoid ICE.")
+    return()
+endif ()
+
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(atomic)
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(gather)
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(gups)
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(launch_latency)
 KOKKOS_ADD_BENCHMARK_DIRECTORIES(stream)
 
-#FIXME_OPENMPTARGET - These two benchmarks cause ICE. Commenting them for now but a deeper analysis on the cause and a possible fix will follow.
-IF(NOT Kokkos_ENABLE_OPENMPTARGET)
+if (NOT Kokkos_ENABLE_OPENMPTARGET)
     KOKKOS_ADD_BENCHMARK_DIRECTORIES(policy_performance)
     KOKKOS_ADD_BENCHMARK_DIRECTORIES(bytes_and_flops)
-ENDIF()
\ No newline at end of file
+else ()
+    # FIXME_OPENMPTARGET - These two benchmarks cause ICE.
+    # Commenting them for now but a deeper analysis on the cause and a possible fix will follow.
+    if (Kokkos_ENABLE_BENCHMARKS)
+        message(WARNING "OpenMPTarget is enabled: disabling 2 benchmarks to avoid ICE.")
```

Comment:
I would't raise a warning for this.  I think the previous version was good enough.

Reply:
Ok, I revert the last commit, and rebase to have an updated SYCL test.